skglm/experimental/reweighted.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+from skglm.estimators import GeneralizedLinearEstimator
+from skglm.penalties import WeightedL1
+
+
+def _L05_objective(coef):
+    return np.sqrt(np.abs(coef))
+
+
+def _L05_derivative(coef):
+    return 1. / (2. * np.sqrt(np.abs(coef)) + np.finfo(float).eps)
+
+
+def _L23_objective(coef):
+    return np.abs(coef) ** (2/3)
+
+
+def _L23_derivative(coef):
+    return 2 / 3 / (np.abs(coef) ** (1/3) + np.finfo(float).eps)
+
+
+def _log_sum_objective(coef):
```

Comment:
Can you create a log sum penalty class?

Reply:
I'd propose to do it in another PR. @Badr-MOUFAD , can you do it?